src/lib/y2storage/clients/inst_disk_proposal.rb

```diff
@@ -97,7 +97,7 @@ def guided_setup(settings)
       end
 
       def expert_partitioner
-        ExpertPartitioner::MainDialog.new.run
+        Y2Partitioner::Clients::Main.run
```

Comment:
That's a good question and IMO shows that the package split logic is kind of broken here.

The 'lib' package requires the package that uses it.

I'd leave it as is for now. More would be out of scope, I think.